Sources/Views/MessageInputBar.swift

```diff
@@ -340,7 +340,7 @@ open class MessageInputBar: UIView {
     private func setupConstraints() {
         
         // The constraints within the MessageInputBar
-        separatorLine.addConstraints(topAnchor, left: leftAnchor, right: rightAnchor, heightConstant: 1)
+        separatorLine.addConstraints(topAnchor, left: leftAnchor, right: rightAnchor)
```

Comment:
Does this change the default appearance?

Reply:
Nope.